luna/gateware/usb/request/standard.py

```diff
@@ -158,6 +160,30 @@ def elaborate(self, platform):
                     # TODO: copy the remote wakeup and bus-powered attributes from bmAttributes of the relevant descriptor?
                     self.handle_simple_data_request(m, transmitter, 0, length=2)
 
+                with m.State('CLEAR_FEATURE'):
+                    # Provide an response to the STATUS stage.
+                    with m.If(self.interface.status_requested):
+
+                        # If our stall condition is met, stall; otherwise, send a ZLP [USB 8.5.3].
+                        # For now, we only implement clearing ENDPOINT_HALT.
+                        stall_condition = \
+                            (self.interface.setup.recipient != USBRequestRecipient.ENDPOINT) | \
+                            (self.interface.setup.value     != USBStandardFeatures.ENDPOINT_HALT)
+                        with m.If(stall_condition):
+                            m.d.comb += self.interface.handshakes_out.stall.eq(1)
+                        with m.Else():
+                            m.d.comb += self.send_zlp()
+
+                    # Accept the relevant value after the packet is ACK'd...
+                    with m.If(self.interface.handshakes_in.ack):
+                        m.d.comb += [
+                            self.interface.clear_endpoint_halt.enable   .eq(1),
+                            self.interface.clear_endpoint_halt.direction.eq(self.interface.setup.index[7]),
+                            self.interface.clear_endpoint_halt.number   .eq(self.interface.setup.index[0:4]),
+                        ]
+
+                        # ... and then return to idle.
+                        m.next = 'IDLE'
```

Comment:
Could we take advantage of the aliases defined above, at the top of `elaborate`?
- `interface` instead of `self.interface`
- `setup` instead of `self.interface.setup`
- `handshake_generator` instead of `self.interface.handshakes_out`